pkgs/misc/uboot/default.nix

```diff
@@ -126,6 +126,48 @@ let
       maintainers = with maintainers; [ bartsch dezgeg samueldr lopsided98 ];
     } // extraMeta;
   } // removeAttrs args [ "extraMeta" ]));
+
+  # Flashing instructions:
+  # dd if=${drv}/u-boot.bin.sd.bin of=$DEV conv=fsync,notrunc bs=512 skip=1 seek=1
+  # dd if=${drv}/u-boot.bin.sd.bin of=$DEV conv=fsync,notrunc bs=1 count=440
+  buildUBootOdroidN2 = fipDirectory: let
+    firmwareImagePkg = fetchFromGitHub {
+      owner = "LibreELEC";
+      repo = "amlogic-boot-fip";
+      rev = "4369a138ca24c5ab932b8cbd1af4504570b709df";
+      sha256 = "sha256-mGRUwdh3nW4gBwWIYHJGjzkezHxABwcwk/1gVRis7Tc=";
+      meta.license = lib.licenses.unfreeRedistributableFirmware;
+    };
+  in
+  assert stdenv.buildPlatform.system == "x86_64-linux"; # aml_encrypt_g12b is a x86_64 binary
```

Comment:
Can you try using https://github.com/angerman/meson64-tools instead?

Hints here:

 - https://github.com/Tow-Boot/Tow-Boot/blob/79acaaba7f44f91c0e0e32cf9062f2cb443575e6/modules/hardware/amlogic/default.nix#L116-L158

Reply:
Not rude to ping me at all, ever 😄 it's always up to me to respond or not. License hmm...

Reply:
Fixed

Reply:
Done :)
Do you want me to add someone else as the meson64-tools package maintainer? It's now an updated version of the  package from tow-boot.